hugo/layouts/docs/single.html

```diff
@@ -10,7 +10,11 @@
                         <h1 class="article__title">{{ .Title }}</h1>
                     </header>
                     <div class="article__content">
-                        {{ .Content }}
+                        {{ if .Params.coming_soon }}
+                            <div class="todo">this page will be coming soon</div>
```

Comment:
> Not sure if we should force content when coming_soon is set to true.

So would this work like `draft` then? If set to true, the content doesn't show? 

I would treat the `disabled` param to mostly work for indexes.  A site visitor would see what types of content is intended to be written but hasn't yet.

Reply:
@Lyoness No it doesn't work together with Draft=true. Hugo will skip all content with draft set to true so it won't show up anywhere. If you want a page to be disabled in the nav you have to set draft to false and disabled to true.

Reply:
@JorindeUsMedia I think the above solution of having a link disabled in the nav, but when clicked on, it says, "TODO" is a good option.  We can tweak later if that doesn't work.

Reply:
@Lyoness So just to summarize: Do you want us to not the markdown show content but show a todo when disabled is to true.
Or do want to have total control over this yourself? Which means we won't do anything in the detailpage itself  when disabled is set to true.

Reply:
>not [show] the markdown show content but show a todo when disabled is to true. 

Let's do this for option and see how it works.

Reply:
@anne-usmedia So for you: Keep what you have in single.html (but use 'disabled' instead of 'coming_soon') but make sure it comes from the translation files.